common/oauthTokenManager.go

```diff
@@ -305,33 +291,7 @@ func (uotm *UserOAuthTokenManager) UserLogin(tenantID, activeDirectoryEndpoint s
 // If refresh token is expired, the method will fail and return failure reason.
 // Fresh token is persisted if access token or refresh token is changed.
 func (uotm *UserOAuthTokenManager) getCachedTokenInfo(ctx context.Context) (*OAuthTokenInfo, error) {
```

Comment:
Can we just delete this if its always returning nil, nil?

Reply:
Done in the latest commit.